### PR TITLE
Vlad

### PR DESCRIPTION
**Pull request by holkwen**

### CODE REVIEW

File: src/main/java/ua/goit/java5/group62/DAO/jdbc/SkillDaoJdbc.java [hunk 1]
```diff
@@ -2,32 +2,110 @@
...
 
 public class SkillDaoJdbc implements SkillDAO {
     @Override
     public int create(Skill skill) {
-        return 0;
+        Connection connection = ConnectionHelper.getConnection();
+        String sql = "INSERT INTO skills VALUES (?, ?)";
+        int result;
+        try (PreparedStatement statement = connection.prepareStatement(sql)) {
+            statement.setLong(1, skill.getId());
```
Comment (AlexPoliteRudenko) [inline src/main/java/ua/goit/java5/group62/DAO/jdbc/SkillDaoJdbc.java]: Why you set long? id type is int.

File: src/main/java/ua/goit/java5/group62/model/Skill.java [hunk 2]
```diff
@@ -1,4 +1,22 @@
 package ua.goit.java5.group62.model;
 
 public class Skill {
+    private long id;
```
Comment (AlexPoliteRudenko) [inline src/main/java/ua/goit/java5/group62/model/Skill.java]: The same. Why it's long?

File: src/main/java/ua/goit/java5/group62/utils/ConnectionHelper.java [hunk 3]
```diff
@@ -1,4 +1,37 @@
...
 
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
 public class ConnectionHelper {
+    private static final String URL = "jdbc:mysql://localhost:3306/it_world_db?useSSL=false";
+    private static final String USERNAME = "root";
+    private static final String PASSWORD = "1111";
```
Comment (AlexPoliteRudenko) [inline src/main/java/ua/goit/java5/group62/utils/ConnectionHelper.java]: In my DB i have another password. And username in other db could differ. How we can handle this problem?

File: src/main/java/ua/goit/java5/group62/DAO/jdbc/SkillDaoJdbc.java (hunk 1 above)
Comment (holkwen) [inline src/main/java/ua/goit/java5/group62/DAO/jdbc/SkillDaoJdbc.java]: Normally all IDs in Java classes are long. Now, indeed, we have the int field in our database. There are two ways: either we make IDs in Java int, too, or we set the ID fields in the database to BIGINT.

File: src/main/java/ua/goit/java5/group62/utils/ConnectionHelper.java (hunk 3 above)
Comment (holkwen) [inline src/main/java/ua/goit/java5/group62/utils/ConnectionHelper.java]: Well, technically we can ask for login data when the program starts.

Comment (AlexPoliteRudenko) [inline src/main/java/ua/goit/java5/group62/utils/ConnectionHelper.java]: I think it's a good solution for this problem.

File: src/main/java/ua/goit/java5/group62/DAO/jdbc/SkillDaoJdbc.java (hunk 1 above)
Comment (AlexPoliteRudenko) [inline src/main/java/ua/goit/java5/group62/DAO/jdbc/SkillDaoJdbc.java]: OK, I'll change fields type in DB

**Pull request merged**